### PR TITLE
fixed errors, moved unused content, added planet, added butchered lore

### PR DESCRIPTION
**Pull request by QmelZ**
hey sh1p how are you

### CODE REVIEW

File: content/blocks/burner.hjson [hunk 1]
```diff
@@ -6,11 +6,12 @@ size: 1
 powerProduction: 0.15
 hasItems: true
 itemCapacity: 10
-itemDuration: 120
+itemDuration: 70
 generateEffect: smelt
-heatColor: 66a8ff
-minItemEfficiency: 0.1
+minItemEfficiency: 0.3
 requirements: [
```
Comment (tezlm) [inline content/blocks/burner.hjson]: just delete this file

Comment (QmelZ) [reply content/blocks/burner.hjson]: no it should be ok to merge

**Comment by tezlm** [comment]
h

**Comment by QmelZ** [comment]
h cool

**Comment by tezlm** [comment]
lmao

**Comment by QmelZ** [comment]
this was very h of us

**Comment by ghost** [comment]
you idea is very good

**Comment by QmelZ** [comment]
yes pankiwi its already in the mod

**Pull request merged**